src/logic/documents/scorecards.js

```diff
@@ -200,6 +204,7 @@ const scorecards = (wcif, rounds, rooms) => {
               localNamesFirst,
               printStations,
               scorecardPaperSize,
+              featured: featuredCompetitorIds.includes(competitor.wcaUserId),
```

Comment:
Let's add a default in `wcif-extensions.js`, otherwise it will crash for competitions without it :)

Reply:
Added!